Resources/Prototypes/Entities/Objects/Consumable/Smokeables/base_smokeables.yml

```
@@ -25,6 +25,28 @@
   components:
   - type: Smokable
     exposeTemperature: 1173.15
+    inhaleAmount: 0.35 # SS220 Cigarettes rework
```

По хорошему тут бы тоже все пометить за один комментарий, чтобы потом если что можно было разобраться
```suggestion
    inhaleAmount: 0.35 # SS220 Cigarettes rework
```